pkg/workceptor/kubernetes.go

```diff
@@ -203,7 +206,12 @@ func (kw *kubeUnit) createPod(env map[string]string) error {
 			return kw.clientset.CoreV1().Pods(ked.KubeNamespace).Watch(kw.ctx, options)
 		},
 	}
-	ev, err := watch2.UntilWithSync(kw.ctx, lw, &corev1.Pod{}, nil, podRunningAndReady)
+
+	ctxWithTimeout := kw.ctx
```

Comment:
sometimes this context is not a timeout, so let's just call it something else
`ctxPodReady` instead of `ctxWithTimeout`

Reply:
I have made the required changes.